science/viking/files/autogen.sh

```diff
@@ -7,40 +7,32 @@ test -n "$srcdir" || srcdir=.
 olddir=`pwd`
 cd "$srcdir"
 
-INTLTOOLIZE=`which intltoolize`
+INTLTOOLIZE=`command -v intltoolize`
```

Comment:
Is this actually necessary for it to work?

Reply:
`autogen.sh` is just a copy of the same file in the github repository. I replaced it, because the previous version was a copy of the same from the 1.7 version.

Reply:
If there's a github repo with better content, why are we using the sourceforge version?

Reply:
They are identical